apps/docs/src/app/core/component-docs/rating-indicator/examples/rating-indicator-base.component.ts

```diff
@@ -0,0 +1,12 @@
+import { Component } from '@angular/core';
+
+@Component({
+    selector: 'app-rating-indicator-base',
+    templateUrl: './rating-indicator-base.component.html',
+    styleUrls: ['./rating-indicator-style.component.scss']
+})
+export class RatingIndicatorBaseComponent {
+    onRatingChanged(event: number): void {
+        console.log(event)
```

Comment:
remove log

Reply:
It's an example. `log` is not really a helpful example, maybe we have a default flow to show event data. If yes, share it with me, please.